packages/create-block/lib/init-block.js

```diff
@@ -32,6 +32,7 @@ async function initBlockJSON( {
 	viewStyle,
 	render,
 	viewModule,
```

Comment:
We don't need `viewModule` anymore in `@wordpress/create-block`. It's been replaced with `viewScriptModule` in all places now.

Reply:
Thanks for the feedback! I would like to submit a follow-up PR.